arches/app/media/js/utils/create-vue-application.js

```diff
@@ -7,16 +7,60 @@ import StyleClass from 'primevue/styleclass';
 import ToastService from 'primevue/toastservice';
 import Tooltip from 'primevue/tooltip';
 
-import Aura from '@primevue/themes/aura';
-
+import { definePreset } from '@primevue/themes';
 import { createApp } from 'vue';
 import { createGettext } from "vue3-gettext";
+import Aura from '@primevue/themes/aura';
 
 import arches from 'arches';
 
+export const ArchesPreset = definePreset(Aura, {
```

Comment:
I'm opening to moving this to its own file, of course.

Reply:
Yep! Please move this to its own file when you get a chance. IMO it should live in `src/arches` but happy to have a discussion around it.

Reply:
We could use cyan instead. [See here](https://primevue.org/theming/styled/)